base.dockerfile

```diff
@@ -44,5 +44,5 @@ EXPOSE 3000
 WORKDIR $APP_BUNDLE_DIR/bundle
 
 # start the app
-ENTRYPOINT ["./entrypoint.sh"]
+ENTRYPOINT [$BUILD_SCRIPTS_DIR/entrypoint.sh]
```

Comment:
I am running it with that change, but you're there's no point in doing that change.  I'll revise the PR.